generators/client/templates/vue/src/main/webapp/app/account/activate/activate.vue.ejs

```diff
@@ -1,18 +1,18 @@
 <template>
-<div>
-  <div class="row justify-content-center">
-      <div class="col-md-8">
-          <h1 v-text="$t('activate.title')">Activation</h1>
-          <div class="alert alert-success" v-if="success">
-              <span v-html="$t('activate.messages.success')"><strong>Your user account has been activated.</strong> Please </span>
-              <a class="alert-link" v-on:click="openLogin" v-text="$t('global.messages.info.authenticated.link')">sign in</a>.
-          </div>
-          <div class="alert alert-danger" v-if="error" v-html="$t('activate.messages.error')">
-              <strong>Your user could not be activated.</strong> Please use the registration form to sign up.
-          </div>
-      </div>
-  </div>
-</div>
+    <div>
```

Comment:
here we're using 4 spaces for indent, and in error.vue.ejs we're using 2 spaces. I think we should be consistent

So maybe we can use prettier for formatting our vue file ? Could we try it ?

Reply:
@pascalgrimaud I think that we could try in an other PR :)

Reply:
@pascalgrimaud I try in this PR ;)

Reply:
To use prettier we have to rename from `.ejs` to `.vue` if there is no ejs template.